velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -111,5 +330,275 @@ TEST_F(RegexFunctionsTest, RegexMatchRegistration) {
   EXPECT_THROW(regexp_extract("[]", "[a[b]]"), VeloxUserError);
 }
 
+TEST_F(RegexFunctionsTest, RegexReplaceRegistration) {
```

Comment:
naming: test names must start with lowercase letter

Reply:
Renamed all rlike, regexp_extract, and regex_replace tests to lowercase.